<launch>
    <arg name="model_name" default="kingfisher_sim"/>
    <arg name="config_pkg" default="$(eval find(arg('model_name') + '_description'))"/>

    <node pkg="tauv_common" type="gate_detector_new" name="gate_detector" output="screen">
        <rosparam command="load" file="$(arg config_pkg)/yaml/gate_detector.yaml" />
        <rosparam param="frame_id">kf/vehicle</rosparam>
        <remap from="color" to="/kf/vehicle/oakd_front/color/image_raw"/>
        <remap from="depth_map" to="/kf/vehicle/oakd_front/stereo/depth_map"/>
        <remap from="camera_info" to="/kf/vehicle/oakd_front/stereo/left/camera_info"/>

    </node>
</launch>
